### PR TITLE
July news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2022-07-22-ropensci-news-digest-july-2022/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### COC place-holder
+
```
Comment (yabellini) [inline content/blog/2022-07-22-ropensci-news-digest-july-2022/index.Rmd:48]:
```suggestion
We are pleased to announce the release of a new version of our Code of Conduct.  Based on the feedback of our community we add greater detail about acceptable and unacceptable behaviors in online settings and we have the first translation of the text to Spanish.

### Committee Changes

We thank [Megan Carter](https://www.esipfed.org/about/people/megan-carter) for serving as independent community member until June, 2022 and welcome back [Kara Woo](https://karawoo.com/) to serve on this role. [Yanina Bellini Saibene](https://ropensci.org/author/yanina-bellini-saibene/) joins the committee as the new rOpenSci Community Manager.

You can read all the details here: https://ropensci.org/blog/2022/07/12/coc-update/

```

File: content/blog/2022-07-22-ropensci-news-digest-july-2022/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### COC place-holder
```
Comment (yabellini) [inline content/blog/2022-07-22-ropensci-news-digest-july-2022/index.Rmd:47]:
```suggestion
### CrOpenSci Code of Conduct Update
```

**Review by yabellini — COMMENTED** [review]
I add a draft of the CoC update information.

File: content/blog/2022-07-22-ropensci-news-digest-july-2022/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+<!-- Before sending DELETE THE INDEX_CACHE and re-knit! -->
+
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### rOpenSci Code of Conduct Update
```
Comment (steffilazerte) [inline content/blog/2022-07-22-ropensci-news-digest-july-2022/index.Rmd:47]:
```suggestion
### rOpenSci Code of Conduct update
```

File: content/blog/2022-07-22-ropensci-news-digest-july-2022/index.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+* [rOpenSci Code of Conduct Update](/blog/2022/07/12/coc-update) by Yanina Bellini Saibene, Mark Padgham, and Kara Woo. Update on Code of Conduct 2022.
+
+
+
+### Tech Notes
+
+* [Evaluating GitHub Activity for Contributors](/blog/2022/07/01/evaluating-github-activity-for-contributors) by Maëlle Salmon. How to evaluate the activity of a GitHub repository as an user or potential contributor. This post was discussed on the [R Weekly Highlights podcast](https://share.fireside.fm/episode/87RSVeFz+17WMAquN?t=1491).
+
+## Use cases
```
Comment (steffilazerte) [inline content/blog/2022-07-22-ropensci-news-digest-july-2022/index.md:126]: Reminder to see if this section needs to be removed if we don't get any new usecases by next Friday

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle! I had to work really hard to find anything to suggest changes to :wink: They're only nit-picky capitalizations and a reminder to check for usecases (which you probably don't need).

I ❤️ learning about pak! I always end up calling my spouse in to learn about these new tricks, thanks!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

File: content/blog/2022-07-22-ropensci-news-digest-july-2022/index.md (hunk 4 above)
Comment (maelle) [reply content/blog/2022-07-22-ropensci-news-digest-july-2022/index.md:126]: Good point! Still nothing but I saw at least one of your tweets encouraging use cases so fingers crossed!

**Comment by maelle** [comment]
@steffilazerte thanks a ton for your review! I was very happy too when I learnt about `pak::pak()` :grin:

**Pull request merged**